openmoc/thisown.i

```diff
@@ -106,6 +106,26 @@
       universes[i][j].thisown = 0
 %}
 
+/* A Lattice owns the memory for each Universe it contains */
+%pythonappend Lattice::setUniverses3D %{
+  # SWIG 3
```

Comment:
I've tried it with SWIG 3 on my mac and SWIG 2 on AWS and it has been working fine in both environments.